src/inmanta/module.py

```diff
@@ -712,7 +712,7 @@ def get_namespace_package_name(cls, module_name: str) -> str:
 
     def install(self, project: "Project", module_spec: List[InmantaModuleRequirement]) -> Optional["ModuleV2"]:
         module_name: str = self._get_module_name(module_spec)
-        if not self.urls:
+        if not self.urls and not project.metadata.pip.use_config_file:
```

Comment:
We could add something to the exception message like "or set the use_config_file project option to true to use the pip config file defined in the PIP_CONFIG_FILE env variable." ?

Reply:
good idea!